lib/src/http/managers/message_manager.dart

```diff
@@ -394,7 +395,7 @@ class MessageManager extends Manager<Message> {
   ///
   /// [raw] must be the inner `message` field from the actual message snapshot
   /// object. See the comment on [MessageReference] for why.
-  MessageSnapshot parseMessageSnapshot(Map<String, Object?> raw) {
+  MessageSnapshot parseMessageSnapshot(Map<String, Object?> raw, {Snowflake? guildId}) {
```

Comment:
Can we pass guildId in the other places we call parseMessageSnapshot?

Reply:
Nah.